tests/e2e/specs/api/DevfileAcceptanceTestAPI.spec.ts

```diff
@@ -3,47 +3,108 @@ import { DevWorkspaceConfigurationHelper } from '../../utils/DevWorkspaceConfigu
+                const applyOutput: ShellString = kubernetesCommandLineToolsExecutor.applyYamlConfigurationAsStringOutput(devWorkspaceConfigurationYamlString);
 
-    test('Create DevWorkspace', async function (): Promise<void> {
-        const devWorkspaceConfigurationYamlString: string = await devWorkspaceConfigurationHelper.getDevWorkspaceConfigurationYamlAsString(devfileContext);
-        const applyOutput: ShellString = kubernetesCommandLineToolsExecutor.applyYamlConfigurationAsStringOutput(devWorkspaceConfigurationYamlString);
+                expect(applyOutput.stdout)
+                    .contains('devworkspacetemplate')
+                    .and.contains('devworkspace')
+                    .and.contains.oneOf(['created', 'configured']);
 
-        expect(applyOutput.stdout)
-            .contains('devworkspacetemplate')
-            .and.contains('devworkspace')
-            .and.contains.oneOf(['created', 'configured']);
+            });
 
-    });
+            test('Wait until DevWorkspace has status "ready"', async function (): Promise<void> {
+                expect(kubernetesCommandLineToolsExecutor.waitDevWorkspace().stdout).contains('condition met');
+            });
 
-    test('Wait until DevWorkspace has status "ready"', async function (): Promise<void> {
-        expect(kubernetesCommandLineToolsExecutor.waitDevWorkspace().stdout).contains('condition met');
-    });
+            test('Check if project was created', function (): void {
+                clonedProjectName = StringUtil.getProjectNameFromGitUrl(devfileSample.link);
+                expect(containerTerminal.ls().stdout).includes(clonedProjectName);
+            });
 
-    suiteTeardown('Delete DevWorkspace', async function (): Promise<void> {
-        kubernetesCommandLineToolsExecutor.deleteDevWorkspace();
-    });
-});
+            test('Check if project files are imported', function (): void {
+                containerWorkDir = containerTerminal.pwd().stdout.replace('\n', '');
+                expect(containerTerminal.ls(`${containerWorkDir}/${clonedProjectName}`).stdout).includes(`devfile.yaml`);
+            });
 
+            test(`Check if build commands returns success`, function (): void {
+                if (devfilesBuildCommands.length === 0) {
+                    Logger.info(`Devfile does not contains build commands.`);
+                } else {
+                    let workingDir: string;
+                    devfilesBuildCommands.forEach((command) => {
+                        Logger.info(`command.exec.commandLine: ${command.exec.commandLine}`);
+                        Logger.info(`command.exec.component: ${command.exec.component}`);
+                        Logger.info(`command.exec.workingDir: ${command.exec.workingDir}`);
+
+                        workingDir = StringUtil.getFullWorkingDirPathExplicit(command.exec.workingDir, containerTerminal);
```

Comment:
Could you try to simplify the code and use pre-defined environment variables in container if execute  bash command `-- bash -c "${commandToExecute}"`, following suggestion from @l0rd ?

Reply:
Updated with method updateCommandEnvsToShStyle(), all tests passed

Reply:
thanks